_data/shaivalmehta27/shaivalmehta27.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Shaival Mehta # No longer than 28 characters
+institution: Gandhinagar Institute Of Technology 🚩 # no longer than 58 characters
+profile_pic: shaivalmehta27.jpg # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
+quote: There will come a time when you'll feel I can't code anymore, just don't stop. Don't give up. Life will fix your bugs automatically. # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.
```

Comment:
Your quote does not meet the character length requirement (100 characters max, current character count: 132).